examples/federated_learning/surface_defect_detection/training_worker/interface.py

```diff
@@ -34,17 +34,18 @@ def build():
         model.add(Conv2D(64, kernel_size=(3, 3),
                          activation="relu", strides=(2, 2),
                          input_shape=(128, 128, 3)))
-        model.add(MaxPooling2D(pool_size=(2, 2)))
-        model.add(Conv2D(32, kernel_size=(3, 3), activation="relu"))
-        model.add(MaxPooling2D(pool_size=(2, 2)))
+        model.add(MaxPooling2D(pool_size=(2, 2), strides=2, padding="same"))
+        model.add(Conv2D(32, kernel_size=(3, 3), activation="relu",
+                         padding="same"))
+        model.add(MaxPooling2D(pool_size=(2, 2), strides=2, padding="same"))
```

Comment:
If this lines is not related to the bug, you should move them to another commit.

Reply:
fix